packages/web-components/src/stories/slider.stories.mdx

```diff
@@ -219,13 +219,68 @@ export const WithAxisLabels = (args, data) => {
 </Canvas>
 
 To display tick marks and labels, make use of the `axis-labels` property. This accepts an array of strings that will
-be evenly distributed across the progress bar. 
+be evenly distributed across the progress bar.
 
 #### Ticks Only
 
 If you only want to show tick marks without labels, make use of the `ticks-only` property. NOTE: you will still need to pass in the array of strings to determine how many
 ticks to show.
 
+### Dual Range
+
+Adding the `min-val` property to the slider will add a second thumb, creating a dual-range slider.
```

Comment:
That's a good point, I always get those mixed up. I'll change the reference to be attribute since what I'm getting at is how the user would use the attribute